src/toolkits/feature_engineering/word_trimmer.cpp

```diff
@@ -204,6 +200,12 @@ void word_trimmer_topk_index_mapping(const gl_sarray& src,
           break;
         }
 
+        case flex_type_enum::UNDEFINED:
+          logstream(LOG_WARNING)
+              << "Skip undefined value in 'word_trimmer_topk_index_mapping'; "
```

Comment:
I don't think the user is going to have any idea what 'word_trimmer_topk_index_mapping' is. I would just say `Skipping undefined values. Consider calling dropna on the feature columns.`

Reply:
I believe this warning will actually go into the server log. So the user won't actually see this.

Reply:
It's still worth fixing the log message for better clarity.

Reply:
anyway, log stream will append src context. Added more user friendly warning.